layouts/shortcodes/python-venv.html

```diff
@@ -0,0 +1,24 @@
+<p>
+    For Python, before we deploy the stack, the following commands need to be
+    run to create a virtual environment, activate it, and install dependencies:
+</p>
+
+<p>Create a virtual environment:</p>
+<div class="highlight">
+    <pre class="chroma"><code class="language-bash" data-lang="bash">$ python3 -m venv venv</code></pre>
```

Comment:
On Windows, this should be `python` instead of `python3`. The default installation of Python on Windows doesn't include a `python3` binary.